bin/security/setupSecurityAccount.sh

```diff
@@ -4,18 +4,17 @@ option="${1}"
 
 BB_ROOT_DIR=$(cd $(dirname "$0"); cd ..; pwd)
 COMPOSE=$BB_ROOT_DIR/$(ls $BB_ROOT_DIR | awk '/docker-compose/ && !/test-tools/')
+MAKEFILE=$BB_ROOT_DIR/../Makefile
```

Comment:
The COMPOSE variable can be removed.
The MAKEFILE path should be $BB_ROOT_DIR/Makefile

Reply:
> I get this error using `$BB_ROOT_DIR/Makefile`
> 
> ```
> make: *** No rule to make target '/home/lenny/edgex/blackbox-testing/bin/Makefile'.  Stop.
> ```
Did you directly run the script (`./bin/security/setupSecurityAccount.sh`) ? I was running tests with bin/run.sh

Reply:
yes.

> Did you directly run the script (`./bin/security/setupSecurityAccount.sh`) ? I was running tests with bin/run.sh

Yes. I will try bin/run.sh also.

Reply:
> The MAKEFILE path should be $BB_ROOT_DIR/Makefile

@jinlinGuan , fixed .